src/modules/Storage.rb

```diff
@@ -5021,6 +5021,7 @@ def AddSubvolRoot(part)
         "var/lib/libvirt/images",
         "var/lib/mailman",
         "var/lib/mariadb",
+        "var/lib/mysql",
```

Comment:
var/lib/mysql is used by openSUSE and SLE's mariadb packages, so the addition is really required

var/lib/mariadb isn't used by any package I can find, so it's kind of useless

But, someday that might change (there's been talk by debian for example to start using /var/lib/mariadb as newer versions are not compatable with mysql) so keeping it around doesn't cost anything